docs/github-webhook.md

```diff
@@ -0,0 +1,90 @@
+# Github webhook Setup for Discord Bot
+
+This documentation details the process of setting up an integration between Discord and GitHub, enabling seamless communication and automation of key notifications directly within your Discord channels. By connecting your GitHub repository with Discord, you'll be able to receive instant alerts about opened `Pull requests`. This integration facilitates real-time collaboration and keeps your team informed about repository changes, improving efficiency and responsiveness to project updates. The integration needs some configs like generate `discord webhook`, setup a `github webhook` in a repository and create `Personal Access Token` in github profile (PAT)
+
+The following keys, filled it with the information of your configuration:
+
+```
+  GITHUB_PR_REVIEW_CHANNEL = DISCORD_CHANNEL_REVIEW
+  GITHUB_ACCESS_TOKEN = YOUR_GITHUB_ACCESS_TOKEN
+```
+
+- **GITHUB_PR_REVIEW_CHANNEL**: Discord channel where the bot will send PR review messages.
+- **GITHUB_ACCESS_TOKEN**: Your access token for authentication.
+
+> [!NOTE]
+> if you want detailed information about this integration process you could check this [doc](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks)
+
+## Steps to Enable Integration Github Webhook
+
+### 1. Generate discord webhook
+
+To enable discord webhook, you need to one integration a service account and obtain webhook url where github will send messages about PRs.
```

Comment:
I think the message is not clear as it is currently, what do you think about something like this?

```suggestion
To enable Discord's Webhook, you need to go to your server settings and obtain Webhook URL where github will send messages about PRs.
```

Or let me know in spanish what was intended to say with this paragraph maybe we can improve the translation

Thanks in advance